README.md

```diff
@@ -78,19 +74,9 @@ The [NCEPLIBS-external documentation](https://github.com/NOAA-EMC/NCEPLIBS-exter
 
 **Obtaining the code**
 
-First, clone the repository:
-
-`git clone https://github.com/NOAA-EMC/NCEPLIBS`
-
-Then, switch to the release branch:
-
-`cd NCEPLIBS`
-
-`git checkout release/public-v1`
-
-Since this project uses submodules they must be initialized by running:
+Clone the repository and its submodules:
 
-`git submodule update --init --recursive`
+`git clone -b ufs-v1.0.0 --recursive https://github.com/NOAA-EMC/NCEPLIBS`
```

Comment:
Should this be `release/public-v1`?

Reply:
According to Arun, the branch will be deleted at some point, but the tag will stay (this PR will be retagged as ufs-v1.0.0).